packages/server/src/api_auth/index.ts

```
@@ -428,29 +427,31 @@ router.post('/magic_login', apiRequestAuthMagicLoginValidator,
                 },
             });
             const response:ApiResponse<ICommon> = {
-                success : true,
+                success: true,
                 status: 200,
-                data:{ token: tokenJWT, isAdmin: false, userId: oldUser.userId, email: oldUser.email }
+                data: { token: tokenJWT, isAdmin: false, userId: oldUser.userId, email: oldUser.email }
             }
             res.status(200).send(response);
             return;
         } catch (error) {
+            // Handle errors and send appropriate response
             if (error instanceof ZodError && !error.isEmpty) {
-                const response:ApiResponse<null> = {
-                    success : false ,
-                    status : 400,
-                    error:'Token and email are required and must be non-empty'
```

We could return the Zod error message.